src/app/actions.ts

```
@@ -9,6 +9,47 @@ import { eq, and, gte, lte, or } from 'drizzle-orm';
+  endDate: string;
+  leaveType: string;
+}): Promise<void> {
+  try {
+    // Get the user details for the request
+    const requestUser = await db
+      .select()
+      .from(users)
+      .where(eq(users.id, updatedRequest.userId))
+      .limit(1);
+
+    if (requestUser[0] && requestUser[0].slackPresenceUpdate && requestUser[0].slackEmail) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)